scripts/build-unix.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# INTRO:
+# This file is used to build ferdium on both x64 and arm-based macos. It also handles any corrupted node modules with the 'CLEAN' env var (set it to 'true' for cleaning)
```

Comment:
Since this version handles linux as well - we should update this line with that info.